addons/google_calendar/utils/google_event.py

```diff
@@ -136,7 +139,8 @@ def owner(self, env):
             return env.user
         elif self.organizer and self.organizer.get('email'):
             # In Google: 1 email = 1 user; but in Odoo several users might have the same email :/
-            return env['res.users'].search([('email', '=', self.organizer.get('email'))], limit=1)
+            org_email = email_normalize(self.organizer.get('email'))
```

Comment:
Xmo would say "unnecessary variable assignment"

Reply:
XMO is well known to write "write only" python code

Reply:
And we want to limit the size of the line

Reply:
You make me very very sad.